flink-libraries/flink-table/src/main/scala/org/apache/flink/table/calcite/FlinkTypeFactory.scala

```diff
@@ -172,45 +172,20 @@ class FlinkTypeFactory(typeSystem: RelDataTypeSystem) extends JavaTypeFactoryImp
     *
     * @param fieldNames field names
     * @param fieldTypes field types, every element is Flink's [[TypeInformation]]
-    * @param rowtime optional system field to indicate event-time; the index determines the index
-    *                in the final record. If the index is smaller than the number of specified
-    *                fields, it shifts all following fields.
-    * @param proctime optional system field to indicate processing-time; the index determines the
-    *                 index in the final record. If the index is smaller than the number of
-    *                 specified fields, it shifts all following fields.
     * @return a struct type with the input fieldNames, input fieldTypes, and system fields
     */
   def buildLogicalRowType(
```

Comment:
I think it's still fine. It creates a row type for the logical (Calcite) plan. 
But I'm fine changing the name if you have a better one.